client/src/views/profile/OrganizationRecoveryPage.vue

```diff
@@ -0,0 +1,309 @@
+            >
+              <ion-icon
+                :icon="checkmarkCircle"
+                class="checked"
+              />
+              {{ $msTranslate('ExportRecoveryDevicePage.subtitles.fileDownloaded') }}
+            </ion-text>
+          </div>
+        </div>
+
+        <div class="recovery-item">
+          <div class="recovery-item-text">
+            <div class="recovery-item-text__title subtitles-normal">
+              <ion-icon :icon="documentLock" />
+              <span>{{ $msTranslate('ExportRecoveryDevicePage.titles.recoveryKey') }}</span>
+            </div>
+            <ion-text class="recovery-item-text__description body">
+              {{ $msTranslate('ExportRecoveryDevicePage.subtitles.keyExplanation') }}
+            </ion-text>
+          </div>
+
+          <div class="recovery-item-download">
+            <a
+              ref="downloadLink"
+              v-show="false"
+            />
+            <div class="recovery-item-download__button">
+              <ion-button
+                @click="downloadRecoveryKey()"
+                id="downloadButton"
+                :disabled="disableKeyDownload"
+                size="default"
+                :fill="recoveryKeyDownloaded ? 'outline' : 'solid'"
+              >
+                <ion-icon :icon="recoveryKeyDownloaded ? reload : download" />
+                {{
+                  recoveryKeyDownloaded
+                    ? $msTranslate('ExportRecoveryDevicePage.actions.downloadAgain')
+                    : $msTranslate('ExportRecoveryDevicePage.actions.download')
+                }}
+              </ion-button>
+            </div>
+            <ion-text
+              class="recovery-item-download__downloaded body"
+              v-show="recoveryKeyDownloaded"
+            >
+              <ion-icon
+                :icon="checkmarkCircle"
+                class="checked"
+              />
+              {{ $msTranslate('ExportRecoveryDevicePage.subtitles.fileDownloaded') }}
+            </ion-text>
+          </div>
+        </div>
+      </div>
+    </template>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { listOwnDevices, OwnDeviceInfo } from '@/parsec';
+import { IonButton, IonIcon, IonText, IonLabel } from '@ionic/vue';
+import { Information, InformationLevel, InformationManager, InformationManagerKey, PresentationMode } from '@/services/informationManager';
+import { I18n, Translatable, askQuestion, Answer } from 'megashark-lib';
+import { checkmarkCircle, document as documentIcon, download, documentLock, reload } from 'ionicons/icons';
+import { inject, onMounted, ref } from 'vue';
+
+const code = '';
+const content = new Uint8Array();
+const downloadLink = ref();
+const recoveryKeyDownloaded = ref(false);
+const recoveryFileDownloaded = ref(false);
+const disableFileDownload = ref(false);
+const disableKeyDownload = ref(false);
+const informationManager: InformationManager = inject(InformationManagerKey)!;
+const orgId = ref('');
```

Comment:
This is never updated. It has been fixed for the 3.3 release and should be on master now. When you rebase, pay attention to the changes made to ExportRecoveryDevicePage.